lib/service/service.go

```diff
@@ -1447,8 +1447,16 @@ func (process *TeleportProcess) initAuthService() error {
 			warnOnErr(tlsServer.Close(), log)
 		} else {
 			log.Info("Shutting down gracefully.")
-			ctx := payloadContext(payload, log)
-			warnOnErr(tlsServer.Shutdown(ctx), log)
+			// NOTE: Graceful shutdown of auth.TLSServer is disabled right now, because we don't
```

Comment:
Yes.  Graceful restart for auth never really worked though (at least not since the watcher system has been around), so its more of a "planned feature" than a re-enabling at this point.